backend/src/backend/routes/users.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+from uuid import UUID
+
+from fastapi import APIRouter, Depends
+from httpx import codes
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+
+from backend.databse import get_session
+from backend.databse.models import User
+
+router = APIRouter()
+
+
+class UserModel(BaseModel):
+    id: UUID
+    created_on: datetime
+
+    class Config:
+        orm_mode = True
+
+
+@router.post("/user", response_model=UserModel, status_code=codes.CREATED)
+async def create_user(session: Session = Depends(get_session)) -> UserModel:
+    """Post this endpoint to create a user."""
+    new_user = User(created_on=datetime.utcnow())
+    session.add(new_user)
+    session.flush()
+    session.refresh(new_user)
+    return UserModel.from_orm(new_user)
```

Comment:
I was a bit annoyed by this ; but I think that's the way to go. Should the commit fail for some reason, that would happen before fastapi returns so we'd never send un-commited data.

Reply:
So we don't need to change it?